src/layer/_components/admin/cwa-admin-header.vue

```diff
@@ -24,4 +24,16 @@ const isNavEnabled = computed({
     $cwa.admin.setNavigationGuardDisabled(newValue)
   }
 })
+
+function clickHandler (e: any) {
+  $cwa.admin.componentManager.resetStackOnClickMiss(e.target as HTMLElement)
```

Comment:
`addToStack` has been setup so the event that could be used was AddToStackWindowEvent which means we do not need to pass some properties and the same `addToStack` function could be used without having to duplicate checks and calls to reset.
I believe if we just used the `addToStack` function we'd basically end up with exactly the same result with no need for additional function/logic.